docs/src/man/man1/mitsub_vfd.1.adoc

```diff
@@ -73,32 +74,32 @@ In this case pins such as up-to-speed, amps, alarm and status bits are not usefu
+_VFD_NAME_**.fwd** (bit, in):: Forward/reverse pin
+_VFD_NAME_**.run** (bit, in):: Run/stop pin
+_VFD_NAME_**.debug** (bit, in):: Set debug mode pin.
 This will print many messages to the terminal.
-[_VFD NAME_]**.monitor** (bit, in):: set monitor mode pin.
+_VFD_NAME_**.monitor** (bit, in):: Set monitor mode pin.
 If false, request-status command will not be sent to VFD.
 Status, amps, power, motor-feedback, and alarm would then not be useful.
-[_VFD NAME_]**.estop* (bit, in):: set E-stop mode pin
+_VFD_NAME_**.estop* (bit, in):: Set E-stop mode pin.
 This will stop the VFD. Restarting requires the run pin to cycle.
-[_VFD NAME_]**.fwd** (bit, out):: up-to-speed status pin Motor is at requested speed within VFD's settings tolerance.
-[_VFD NAME_]**.alarm** (bit, out):: alarm status pin
-[_VFD NAME_]**.motor-cmd** (float, in):: The requested motor speed, is Hertz
-[_VFD NAME_]**.motor-fb** (float, out):: The motor feedback speed (from VFD) in Hertz (Hz)
-[_VFD NAME_]**.motor-amps** (float, out):: The motor current, in amps
-[_VFD NAME_]**.motor-power** (float, out):: The motor power
-[_VFD NAME_]**.scale-cmd** (float, in):: Motor command's scale setting defaults to 1
-[_VFD NAME_]**.scale-cmd** (float, in):: Motor command's scale setting defaults to 1
-[_VFD NAME_]**.scale-cmd** (float, in):: Motor command's scale setting defaults to 1
-[_VFD NAME_]**.stat-bit-0** (bit, out):: raw status bit
-[_VFD NAME_]**.stat-bit-1** (bit, out):: raw status bit
-[_VFD NAME_]**.stat-bit-2** (bit, out):: raw status bit
-[_VFD NAME_]**.stat-bit-3** (bit, out):: raw status bit set the VFD so this is motor-at-speed status
-[_VFD NAME_]**.stat-bit-4** (bit, out):: raw status bit
-[_VFD NAME_]**.stat-bit-5** (bit, out):: raw status bit
-[_VFD NAME_]**.stat-bit-6** (bit, out):: raw status bit
-[_VFD NAME_]**.stat-bit-7** (bit, out):: raw status bit Set the VFD so this in the alarm bit
+_VFD_NAME_**.fwd** (bit, out):: Up-to-speed status pin Motor is at requested speed within VFD's settings tolerance.
+_VFD_NAME_**.alarm** (bit, out):: Alarm status pin
+_VFD_NAME_**.motor-cmd** (float, in):: The requested motor speed, in Hertz (Hz)
+_VFD_NAME_**.motor-fb** (float, out):: The motor feedback speed (from VFD) in Hertz (Hz)
+_VFD_NAME_**.motor-amps** (float, out):: The motor current, in amperes (A)
+_VFD_NAME_**.motor-power** (float, out):: The motor power
+_VFD_NAME_**.scale-cmd** (float, in):: Motor command's scale setting defaults to 1
+_VFD_NAME_**.scale-cmd** (float, in):: Motor command's scale setting defaults to 1
+_VFD_NAME_**.scale-cmd** (float, in):: Motor command's scale setting defaults to 1
+_VFD_NAME_**.stat-bit-0** (bit, out):: Raw status bit
+_VFD_NAME_**.stat-bit-1** (bit, out):: Raw status bit
+_VFD_NAME_**.stat-bit-2** (bit, out):: Raw status bit
+_VFD_NAME_**.stat-bit-3** (bit, out):: Raw status bit. Set the VFD so this is motor-at-speed status. FIXME: Unclear - is the signal set or is the wiring to be set in a particular way?
```

Comment:
Could some good soul please help with a patch that improves the wording of this description?

Reply:
You have to configure the vfd so that this status bit is mapped to the function "up to frequency"


See https://dl.mitsubishielectric.com/dl/fa/document/manual/inv/sh060014/sh060014engb.pdf

page 458 and pages 61/62:

![grafik](https://github.com/user-attachments/assets/1f9a2396-8530-4296-8eef-065baa6d3283)

![grafik](https://github.com/user-attachments/assets/795092e7-7498-40d7-8541-511917337630)

![grafik](https://github.com/user-attachments/assets/29923cab-42b8-4f65-a054-e2c705af44f7)